packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -56,12 +56,18 @@ export default decorate([
           name,
           options = remote.options || '',
           password = remote.password,
-          path = remote.path,
           port = remote.port,
           proxyId = remote.proxy,
           type = remote.type,
           username = remote.username,
+          parsedPath,
+          bucket = parsedPath && parsedPath.split('/')[0],
+          directory = parsedPath && parsedPath.split('/')[1],
```

Comment:
Please use explicit comparisons.

I don't know what values can `parsedPath` be, is it `false`, `undefined` or empty string?

If it can be not a string, how will that work with the concatenation line 69?